### ISSUE TITLE
make package more python-like

### ISSUE THREAD

**Issue by mrayson** [TIDE-ITRH/speccy#5]
Add

- [ ] setup.py
- [ ] move main code into a sub-folder called speccy
- [ ] make __init__.py that imports sick_tricks as *
- [ ]

**Comment by mrayson** [comment]
@astfalckl I am happy to do the first few steps here to get the ball rolling?

**Comment by astfalckl** [comment]
Yeah go for it, can you just show me how you did it all when we're both in next?
